src/components/structures/MatrixChat.js

```diff
@@ -1183,10 +1186,6 @@ export default React.createClass({
      * @param {string} teamToken
      */
     _onLoggedIn: async function(teamToken) {
-        this.setState({
```

Comment:
Given what we learned from the test failures and looking at this code again with the knowledge that we have to set the logged in state in all branches, I feel like it's actually clearer to read as it was. What do you think about reverting all changes to this function? Sorry for all the distractions of the ensuing test failures yesterday... 😓

Reply:
I'll replace it with the safer option, but sure.